### PR TITLE
ISSUE-124 BlobId list Maven

### PR DESCRIPTION
**Pull request by vttranlina**

### CODE REVIEW

**Comment by chibenwa** [comment]
No also with the java code and tests in that maven module :-)

File: tmail-backend/blob/blobid-list/pom.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>blob-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>blob-s3</artifactId>
+        </dependency>
```
Comment (chibenwa) [inline tmail-backend/blob/blobid-list/pom.xml:44]: Remove this dependency

File: tmail-backend/blob/blobid-list/pom.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>blob-memory</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>blob-cassandra</artifactId>
+            <scope>test</scope>
```
Comment (chibenwa) [inline tmail-backend/blob/blobid-list/pom.xml:59]: Remove this dependency

File: tmail-backend/blob/blobid-list/pom.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            <artifactId>guice</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.projectreactor</groupId>
+            <artifactId>reactor-scala-extensions_${scala.base}</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.james</groupId>
+            <artifactId>james-server-guice-configuration</artifactId>
+        </dependency>
```
Comment (chibenwa) [inline tmail-backend/blob/blobid-list/pom.xml:72]: Remove this dependency

It look suspicious to me to rely on server-guice* dependencies in implementation modules.

**Review by LanKhuat — COMMENTED** [review]
You should arrange the dependencies in alphabetical order.

**Comment by chibenwa** [comment]
Where is the code within this module???

**Comment by vttranlina** [comment]
> Where is the code within this module???

You can follow it at pr https://github.com/linagora/tmail-backend/pull/145

**Comment by chibenwa** [comment]
> You can follow it at pr #145

Ok, then I close that one.

**Pull request closed without merging**